app/views/static/sandbox.scala.html

```diff
@@ -0,0 +1,28 @@
+@*****************************************************************************
+ * Page for testing XPath/XQuery expressions against target resources        *
+ * Useful for authoring Finders                                              *
+ * Copyright (c) 2015 MIT Libraries                                          *
+ *****************************************************************************@
+@(sbForm: Form[(String, String)], results: List[String])@*(implicit hubContext: HubContext)*@
+@import helper._
+@layout.main("Sandbox - TopicHub") {
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+      @form(routes.Application.testExpression()) {
+        @textarea(sbForm("expression"), 'rows -> 8, 'cols -> 40)
+        @inputText(sbForm("resourceUrl"), 'size -> 50)
+        <input type="submit" value="Test Expression">
+      }
+      <br/>
+      <a href="@routes.Application.sandbox" class="btn btn-primary">Start Over</a>
```

Comment:
From a ux perspective, this is confusing as heck to have the primary button reset the form. That said, I don't think that matters for this form so I'd leave it like this for this PR.